### DISCUSSION TITLE
Is there a way to use Free RASP Flutter in GitHub Actions or in other CI/CD Solutions

### DISCUSSION THREAD

**Discussion by Killpit** [talsec/Free-RASP-Community#53] [Q&A]
Hello, 

What are the capabilities of this tool in security scans and and whether we can integrate them in GitHub Actions and other CI/CD pipelines for security scanning?

**Comment by SirionRazzer** [comment]
Hi @Killpit ,
I assume there is some misunderstanding. The freeRASP is not a MAST tool for static security testing. Can you suggest your intended use-case?
Best regards,
Tomáš Soukal from Talsec
